htdocs/expedition/shipment.php

```diff
@@ -623,6 +623,7 @@
 			$i = 0;
 			print '<thead>';
 			print '<tr class="liste_titre">';
+			print '<th>'.$langs->trans("Rank").'</th>';
```

Comment:
To follow other document, can you add the condition 
` if (getDolGlobalString('MAIN_VIEW_LINE_NUMBER')) {`

Reply:
ok it is fixed